### DISCUSSION TITLE
Any support for image generation models? (e.g. Stable Diffusion)

### DISCUSSION THREAD

**Discussion by ClementNerma** [janhq/jan#3056] [Get Help] [answered]
Hi there!

I've been using Jan for quite a while and it's become my main chat app at this point.

I was wondering though if there was any support at all for image generation models such as Stable Diffusion. I didn't see anything in the docs and I can't seem to find generation models in the hub.

So, are these supported? Or is the support coming sooner or later? Or is it an explicit non-goal of the project?

Thanks!

**Comment by kalle07** [comment]
at the moment Jan dosnt support ( i dont now if its a goal)

try koboldcpp for that  ;)
for me the image generation is too complex (like a1111) to merge it in an LLM software, if than is a link the best so that the image is called by a prompt from LLM  . . . 

for documents still GPT4ALL is best also for other quantisation

**Reply by ClementNerma** [reply]
I see. Thanks for your answer! :)

**Comment by ux-han** [comment, marked as answer]
See here #3524
